style.css

```diff
@@ -0,0 +1,114 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&family=Inter:wght@300;400;700&display=swap");
+
+* {
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0;
+}
+
+header {
+  display: flex;
+  background-color: #fff;
+  width: 100%;
+  height: 90px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  display: flex;
+  color: #5969fa;
+  line-height: 20px;
+  text-align: center;
+  margin-left: 24px;
+}
+
+header > .logo {
+  font-size: 18px;
+}
+
+.union {
+  margin-right: 24px;
+  display: flex;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+  font-family: 'Poppins', sans-serif;
+  background-image: url(./images/Background.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: 0 72px;
+  padding: 24px;
+}
```

Comment:
- [Optional] It would be nice if you could align the background image properly in the headline to make it look similar to the [figma] 😄 (https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 

Figma Design             |  Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/60618877/177525074-be8b1182-a4b1-482b-af9c-a775132a0b39.png)  |  ![image](https://user-images.githubusercontent.com/60618877/177525197-718a7df3-491e-453a-ac5c-ad0f169eed2a.png)